Assets/_Scripts/Managers/HighScoreManager.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class HighScoreManager : Singleton<HighScoreManager>
+{
+
+    private int _highScore;       
+
+
+    // Start is called before the first frame update
+    protected override void Awake()
+    {
+        base.Awake();
+    }
+
+    public int GetHighScore(int _score)
+    {
+        if (_score > _highScore)
+        {
+            _highScore = _score;
+        }
```

Comment:
It's dangerous / misleading for a `GetHighScore` method to also set the high score.

This should be split into two separate methods: a getter and a setter.

The `GetHighScore` should be called by the `CanvasManager` to display the correct text and the `SetHighScore` should be called by the `GameManager` on win to set the high score if applicable.